NightscoutServiceKitTests/BolusRemoteNotificationTestCase.swift

```diff
@@ -0,0 +1,59 @@
+//
```

Comment:
The tests in the next few files just take some data samples and ensures I can still create notifications from it.